src/io/buf_read/mod.rs

```diff
@@ -226,6 +229,57 @@ extension_trait! {
                 read: 0,
             }
         }
+
+        #[doc = r#"
+            Returns a stream over the contents of this reader split on the byte byte.
```

Comment:
Looks like here is a duplicated word of 'byte'.

Reply:
Changed it to byte `byte` to match std's docs: https://doc.rust-lang.org/std/io/trait.BufRead.html#method.split